src/test/java/showcase/neodymium/tests/shadowdom/CheckboxTest.java

```diff
@@ -34,6 +35,8 @@
         
     @Test
     @Description(value = "Showcase for nested shadow DOM.")
+    @Browser("FF_1024x768")
+    @Browser("Chrome_1024x768")
```

Comment:
```suggestion
```

You don't need to add these as the browser should be controller by `AbstractTest`. And since we don't have a Firefox configured there, I would not add it anywhere.